index.html

```diff
@@ -204,9 +203,70 @@ <h2 class="headings">Tonics</h2>
   <a href="#">See projects</a>
 </p>
 
-</div>
+</section >
 
  <div class="card-works">
```

Comment:
- [ ] As requested by the previous reviewer kindly consider sticking to the Figma design as much as possible check on the image below and consider fixing the wrong design before you request another review doing so will make your project meet the design requirements
- your design 👇 
<img width="392" alt="Screen Shot 2023-05-05 at 18 34 30" src="https://user-images.githubusercontent.com/60146030/236516012-584721d9-201d-46a0-8682-f685829c382d.png">
  the borders and the top padding are not exactly like the one given below 

- Figma design 👇 
<img width="309" alt="Screen Shot 2023-05-05 at 18 37 18" src="https://user-images.githubusercontent.com/60146030/236516390-9e6261ed-a81e-4942-9fa2-2368ae8f06ba.png">

Reply:
about the margin, i am to work on it.
here is your comment " Kindly as requested by the previous reviewer consider wrapping the entire section in a section tag not wrapping each work card in a section tag "  .. now at this part you are telling me that there should be borders, quick question how is that to happen if i am to have one card-works section  having the entire content..

Reply:
i mean how just how

Reply:
but you can clearly see the figma template each and every part has a border . now if i am to have a single section tag to contain all card will it be possible to have the exact replica . suppose i was to add divs inside the section tag to enable having card .then i believe it won't be able to produce the exact template . in other words what you are telling me is that section tags are to be used in a limit.. just one to contain the whole card